src/Prometheus/Histogram.php

```diff
@@ -12,27 +12,33 @@ class Histogram extends Collector
     const TYPE = 'histogram';
 
     /**
-     * @var array|null
+     * @var mixed[]|null
      */
     private $buckets;
 
     /**
-     * @param Adapter $adapter
-     * @param string $namespace
-     * @param string $name
-     * @param string $help
-     * @param array $labels
-     * @param array $buckets
+     * @param Adapter    $adapter
+     * @param string     $namespace
+     * @param string     $name
+     * @param string     $help
+     * @param string[]   $labels
+     * @param mixed[]|null $buckets
```

Comment:
I'm not quite sure if `mixed` is the correct type here. 

I looked into the official go client and there it is always a float: https://github.com/prometheus/client_golang/blob/47cfdc9bb8ceaf18b12a4b7ba778b06e12d1a97a/prometheus/histogram.go#L63

So maybe we should also use a float here?